tests/tutorials/test_format.py

```diff
@@ -14,7 +14,7 @@
 ]
 
 docstring_start_pattern = re.compile(r'# %% \[markdown\]\n"""\n#(?: .*:)? \d+\. .*\n(?:\n[\S\s]*)?"""(?:  # .*)?\n')
-comment_start_pattern = re.compile(r'# %% \[markdown\]\n# #(?: .*:)? \d+\. .*\n#(?:\n# [\S\s]*)?')
+comment_start_pattern = re.compile(r"# %% \[markdown\]\n# #(?: .*:)? \d+\. .*\n#(?:\n# [\S\s]*)?")
```

Comment:
Don't knkw honestly, but that's how linter sees it.
I think that's because in the line above we have double quotes in the string so we would have to escape them.